assets/src/js/admin/onboarding-wizard/components/dismiss-link/style.scss

```diff
@@ -0,0 +1,15 @@
+.give-obw-dismiss-link {
+	font-family: 'Open Sans', Arial, Helvetica, sans-serif;
+	color: #666;
+	font-weight: 500;
+	font-size: 16px;
+	line-height: 22px;
+	margin-top: 42px;
+	padding: 6px;
```

Comment:
I'm not sure why, but the dismiss link padding is forcing a horizontal scroll bar on the Features Step. Same for the Add-ons screen.

With padding:
![Screenshot from 2020-08-12 21-01-51](https://user-images.githubusercontent.com/10858303/90082978-35674700-dcdf-11ea-9930-6f7f1ccbce45.png)

Without padding:
![Screenshot from 2020-08-12 21-02-06](https://user-images.githubusercontent.com/10858303/90082981-37c9a100-dcdf-11ea-8bf1-8a0ac26f9136.png)

Reply:
May be resolved by https://github.com/impress-org/givewp/pull/5077#discussion_r469948424